components/typography/typography.js

```diff
@@ -55,31 +53,38 @@ if (!head.querySelector('#d2l-typography-font-face')) {
 			font-size: 0.95rem;
 			font-weight: 400;
 			line-height: 1.4rem;
-			/*@apply --d2l-font-custom;*/
 		}
 
-		/*
 		.d2l-typography .d2l-body-standard {
-			@apply --d2l-body-standard-text;
+			font-size: 0.95rem;
+			font-weight: 400;
+			line-height: 1.4rem;
```

Comment:
Hmm, so these are interesting. Not sure what to do about them eh, they're kind of redundant now?

Reply:
I was thinking that too.  They are almost the same except that these ones scope the selectors with `d2l-typography` and `vui-typography`, and there's also a selector for `p`.   Currently BSI uses the Sass (which is the same as this) plus a few extra overrides for layout/whitespace - I think eventually we'd switch it to include this.